doc/testing_tools/testing_floating_points.qbk

```diff
@@ -135,29 +140,42 @@ inequations [link equ2 (2)] and [link equ3 (3)] where all underflow, overflow co
 || abs(u - v)/abs(v) <= epsilon; // (5)
 ``
 
+The above however, will not work when `v` or `u` is too close to zero. In such cases the solution is to resort to [link equ1 (1)]. 
+
 [h3 Implementation]
```

Comment:
I added this part there when I fixed the support for multiprecision, but now I think it should go to another section since we are under "theory".